vue.config.js

```diff
@@ -1,21 +1,12 @@
 module.exports = {
   crossorigin: 'anonymous',
   devServer: {
-    disableHostCheck: true
+    allowedHosts: 'all',
   },
   pluginOptions: {
     express: {
       shouldServeApp: true,
-      serverDir: './server'
-    }
+      serverDir: './server',
+    },
```

Comment:
Another unnecessary changes. Why do you process this project with prettier?

Reply:
Hm. This change I didn't missed. I thought these were improvements.
Removed.